package/src/components/AttachmentPicker/components/AttachmentPickerItem.tsx

```diff
@@ -70,14 +70,16 @@ const AttachmentVideo: React.FC<AttachmentVideoProps> = (props) => {
           Alert.alert('Maximum number of files reached');
           return files;
         }
+        // Since Expo MediaLibrary doesn't return the mimetype of the image/video, we have to derive the mimeType.
+        const mimeType = lookup(asset.filename) || 'multipart/form-data';
```

Comment:
if expo is the only that doesnt return mime type why is it being done for both here ?

Reply:
as per expo docs, mimetype is optionally present in the asset type 

https://docs.expo.dev/versions/latest/sdk/document-picker/#documentpickerasset

Reply:
also present in older versions

https://docs.expo.dev/versions/v47.0.0/sdk/document-picker/#documentresult

Reply:
If you read the comment, I am not saying expo doesn't do that. `Expo MediaLibrary` doesn't do that, `Expo DocumentPicker` does. In our SDK we don't do lookup for the code for the document picker or file picker, we do it for the image picker. @santhoshvai

Reply:
I thought its related to document picker, because you said that this was broken by the docu,ent picker versions

Reply:
No, what I meant was the previous PR of the document picker that was merged yesterday caused this issue of video upload because mime-type was compulsory.

Reply:
There's a better way to fix this and that we discussed yesterday, handling lookup in the expo-package and native-package and sending mimetype through the package itself